amass/sources/twitter.go

```diff
@@ -100,7 +100,22 @@ func (t *Twitter) executeQuery(domain string) {
 	}
 
 	for _, tweet := range search.Statuses {
-		for _, name := range re.FindAllString(tweet.Text, -1) {
+
+		// Urls in the tweet body
+		for _, url_entity := range tweet.Entities.Urls {
```

Comment:
Variables with multiple words should be camel-cased (`urlEntity`). Alternatively, you could just go with `entity`

Reply:
Agreed. Thanks for the review!